Library/Homebrew/extend/ENV/super.rb

```diff
@@ -164,7 +164,11 @@ def determine_isystem_paths
   end
 
   def determine_include_paths
-    keg_only_deps.map { |d| d.opt_include.to_s }.to_path_s
+    paths = keg_only_deps.map { |d| d.opt_include.to_s }
+    if compiler == :llvm_clang && Formula["llvm"].installed?
```

Comment:
I reckon this should be conditional to `needs :openmp` being used but CC @mistydemeo @ilovezfs for thoughts around this stuff